content/docs/command-reference/params/index.md

```diff
@@ -0,0 +1,140 @@
+# params
+
+A set of commands to manage, and display project parameters:
+[diff](/doc/command-reference/params/diff).
```

Comment:
I see you updated this, thanks. But there's a plan to add those subcommands soon? Otherwise this description could be different, more like in https://dvc.org/doc/command-reference/cache. Mabe:

```
Contains a helper command to show changes in parameters defined with the `-p`
option of `dvc run`: [diff](/doc/command-reference/params/diff).
```

Reply:
I've changed the sentence that you pointed to.
I see the analogy with cache. But in case of params it makes sense to keep a separate page for parameter description since there is not enough room for that in the `dvc run` page when params are supposed to be defined. Let's keep a whole page. Also, we might add more commands.